Source/BoundaryConditions/PML.cpp

```diff
@@ -783,7 +783,7 @@ void
 PML::Exchange (MultiFab& pml, MultiFab& reg, const Geometry& geom,
                 int do_pml_in_domain)
 {
-    BL_PROFILE("PML::Exchange");
+    WX_PROFILE("PML::Exchange");
```

Comment:
So `WX` is how we abbreviate ourselves? :D 
Probably `WARPX_` does not harm as well, `BL_` is confusing enough :P

Reply:
sure, did that.